packages/nextjs-template/pages/notes/[id].tsx

```diff
@@ -16,16 +17,21 @@ import {
 
 export default DendronNotePage;
 
-export const getStaticPaths: GetStaticPaths = getNotePaths;
+export const getStaticPaths: GetStaticPaths<DendronNotePageParams> =
+  getNotePaths;
 
-export const getStaticProps: GetStaticProps = async (
-  context: GetStaticPropsContext
+export const getStaticProps: GetStaticProps<
```

Comment:
See the following URL for a typing explanation.
https://www.vitamindev.com/next-js/getstaticprops-getstaticpaths-typescript/